hoomd/SignalHandler.cc

```diff
@@ -7,16 +7,14 @@
 #include <signal.h>
 #include "SignalHandler.h"
 #include <iostream>
+#include "string.h"
```

Comment:
According to [this post](https://stackoverflow.com/questions/8380805/difference-between-string-h-and-cstring), it's probably better to use `<cstring>` in C++. `<cstring>` _must_ place the functions in the standard namespace and _may_ place the functions in the global namespace, but that's not going to cause any differences here since you have `using namespace std;` below this.
```suggestion
#include <cstring>
```